packages/nouns-contracts/contracts/governance/NounsDAOInterfaces.sol

```diff
@@ -865,6 +566,39 @@ contract NounsDAOStorageV3 {
         bool executeOnTimelockV1;
     }
 
+    struct ProposalCondensedV2 {
```

Comment:
@eladmallel the naming here is a bit confusing, because `ProposalCondensedV2` is actually older than `ProposalCondensed`, right?
any ideas how to make more intuitive?

Reply:
maybe a good solution is to rename the latest condensed struct to something like `ProposalCondensedV4`?